integration_test/third_party_apps_data/applications/hadoop/debian_ubuntu/install

```diff
@@ -0,0 +1,144 @@
+set -e
+
+sudo apt-get update
+sudo DEBIAN_FRONTEND=noninteractive apt-get install -y default-jdk default-jdk-headless wget
+
+sudo mkdir -p \
```

Comment:
These scripts are nearly identical, can we refactor to share common code?

Reply:
So, what is actually copied into the environment? is it the whole hadoop folder?

Reply:
I don't know the answer to that :/
@martijnvans perhaps?

Reply:
I was thinking about something similar. It would likely require some refactoring of the testing infra though to support a shared snippet.

```
- hadoop
   - install
      - common
      - debian_ubuntu
      - centos_rhel
      - sles
```

A follow up PR for this refactoring is better IMO instead of trying to doing that in this PR. So that this PR gets unblocked first, and we could keep the refactoring separate from a new feature.

Reply:
ling is right, we'd need to introduce some kind of "common" script that gets run for all platforms. Such a change should be pretty easy.

Reply:
Cool. I'll leave this thread unresolved for easy reference (GitHub UI makes it hard to find resolved comments). It does not block this PR from being merged.